src/device-registry/controllers/create-airqloud.js

```diff
@@ -158,6 +221,10 @@ const createAirqloud = {
 
   findSites: async (req, res) => {
     try {
```

Comment:
Temporarily disabling the endpoint for finding Sites within any given AirQloud. Done due to failure to resolve a runtime error.